ts/packages/ts/src/ConfigSchema.ts

```diff
@@ -56,6 +56,11 @@ export abstract class ConfigSchema implements IConfigSchema {
           validator.isHash(value, 'sha512'),
         Country: ({ value }) => validator.isISO31661Alpha2(value) || validator.isISO31661Alpha3(value),
         Currency: ({ value }) => validator.isISO4217(value),
+        DockerImage: ({ value }) => 
+          // eslint-disable-next-line no-useless-escape
+          /^(?:(([a-z0-9]|[a-z0-9][a-z0-9\\-]*[a-z0-9])\.)+([a-z0-9]|[a-z0-9][a-z0-9\\-]*[a-z0-9])(:[0-9]+\/)?(?:[0-9a-z-]+[/@])(?:([0-9a-z-]+))[/@]?(?:([0-9a-z-]+))?(?::[a-z0-9\\.-]+)?|([^:\/?#\s]+):\/\/(?:([^@\/?#\s]+)@)?([^\/?#\s]+)?(?:\/([^?#\s]*))?(?:[?]([^#\s]+))?\S*)$/gm.test(
```

Comment:
@Sadaf-A, did you craft this regex pattern yourself or draw inspiration from a source? Could you please provide a comment with either a link to a tester like https://regex101.com/ or the direct source link?

Reply:
@rannn505 I took reference from the `StackOverflow` link that you had provided in the issue itself.
I've updated the `RegExp` and below is the link for testing.
https://regex101.com/r/egVJW9/1 
Thank you,
Sadaf